gmp/gvm_connection.py

```diff
@@ -959,23 +961,34 @@ def __init__(self, **kwargs):
                                    '/usr/local/var/run/gvmd.sock')
         self.shell_mode = kwargs.get('shell_mode', False)
         self.timeout = kwargs.get('timeout', 60)
+        self.read_timeout = kwargs.get('read_timeout', 30)
```

Comment:
Not sure if 30 seconds is big enough. I had/have the same timeout in gsa and we have seen longer requests in the wild. E.g. get_report/get_result can take a while.

Reply:
I would also change this code to use module constants like
```python
DEFAULT_READ_TIMEOUT = 30 # in seconds
DEFAULT_TIMEOUT = 60 # in seconds
```